mreg/signals.py

```diff
@@ -298,3 +299,13 @@ def update_hosts_when_zone_is_added(sender, instance, created, **kwargs):
                 continue
             h.zone = instance
             h.save()
+
+@receiver(pre_save, sender=Host)
+def hosts_in_a_delegation_shall_get_zone_None(sender, instance, raw, using, update_fields, **kwargs):
```

Comment:
Will slow down things massively. Only needed for new instances or updated name, though not sure if it should be done as a dedicated signal at all.